pyproject.toml

```diff
@@ -10,7 +10,6 @@ repository = "https://github.com/ersilia-os/ersilia"
 documentation = "https://ersilia.io/model-hub"
 keywords= ["drug-discovery", "machine-learning", "ersilia", "open-science", "global-health", "model-hub", "infectious-diseases"]
 classifiers=[
-        "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
```

Comment:
How about added support for python 3.11 here since you've already added it in your code above

Reply:
so you mean I should add  python version 3.11 right?

Reply:
While you are at it, please mention 3.12 as well, thank you!

Reply:
Sure please on it